docs/output-logstash.asciidoc

```diff
@@ -106,31 +106,26 @@ This plugin supports the following configuration options plus the <<plugins-{typ
 |=======================================================================
```

Comment:
Scrolling up to the top, this page needs

~~~ diff
@@ -19,1 +19,1 @@
-=== Logstash input plugin
+=== Logstash output plugin
~~~

Reply:
Ah, I knew it, failed to update somehow. 124L is also similar.